modules/order/src/Entity/Order.php

```diff
@@ -422,6 +445,43 @@ public function preSave(EntityStorageInterface $storage) {
   }
 
   /**
+   * {@inheritdoc}
+   */
+  public function collectAdjustments() {
```

Comment:
I know this is what @bojanz wanted, but it feels weird to have this here and then the next change where we don't just do `::getAdjustments()` directly. This feels like it belongs in a service to get the adjustments and their type metadata